docs/creating-a-single-component-application-with-odo.md

```diff
@@ -122,7 +120,7 @@ You can modify your application code and have the changes applied to your applic
 
 # Adding storage to the application components
 
-Persistent storage keeps data available between restarts of odo. Use the `odo storage` command to add persistent data to your application. Examples of data that must persist include database files, dependencies, and build artifacts, such as a `.m2` Maven directory.
+Use the `odo storage` command to add persistent data to your application. Examples of data that must persist include database files, dependencies, and build artifacts, such as a `.m2` Maven directory.
```

Comment:
`Examples of data that is usually persisted`?

Reply:
maybe explain what does it mean when you say `persistent`, like is that when a developer wants to keep their data safe between `odo push`?

Reply:
The way this works is we have to fix this in the `adoc` file in the master branch. This PR (or any PR on `gh-pages` branch for that matter) is just a script that converted these adoc to md. So it updates the existing md files if there's any change in master branch. Or, like in this PR, we add new doc.

To address your concern, all we can do at the moment is open a new issue in the repo and fix it. We need to keep doc review out of the scope of PR review for gh-pages. :disappointed: